docs/setup/hooks.md

```diff
@@ -299,3 +314,5 @@ It should look like `https://api.relay.svix.com/api/v1/play/receive/<Random_Gen_
 in the Svix endpoint you provided. You can also check the `Actions` tab in the lakeFS UI for more details.
 
 ![Setup]({{ site.baseurl }}/assets/img/svix_play.png)
+
+---
```

Comment:
Think you need to remove this one, because it will look like we need to format all our documentation this way.

Reply:
The reason I added it was because the foot notes looked like they belong to the section above them and it was very confusing. Do you have any suggestion on how to fix it?